cmd/cli.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+
+	app := &cli.App{
+		Commands: []*cli.Command{
+			{
+				Name:    "get",
+				Aliases: []string{"c"},
```

Comment:
what's this?

Reply:
this is a get command. to get a list of supported types. Alias should probably be g not c though.

Reply:
```
get typego run cli.go get key types
[Ed25519 X25519 secp256k1 P-224 P-256 P-384 P-521 RSA]
```